sdk/keyvault/ci.yml

```diff
@@ -35,5 +35,7 @@ stages:
       ServiceDirectory: keyvault
       CtestRegex: azure-security-keyvault
       Artifacts:
-        - Name: azure-security-keyvault
-          Path: azure-security-keyvault
+        - Name: azure-security-keyvault-common
```

Comment:
What form will these take? We explicitly have been avoiding any sort of "core" or "common" module across any language being exposed. In .NET, for example, it's a shared source project - effectively, all source files are just statically linked in. There's no output for that project. If/when these are vcpkg-compatible, perhaps that makes sense, but "common" itself should not otherwise be referenceable (ideally).

Reply:
Yes, I saw it is called `Shared` in .NET
This common package is basically the same as the Shared one.  I picked the `common` name to be in sync with the another current SDK package (storage) which is `azure-storage-common`.
However, this was commented some days ago and the team has decided to use `shared` name.  So this will be upated soon (maybe in a next PR)